frontend/packages/console-app/src/components/modals/service-binding/CreateServiceBindingForm.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { TextInputTypes, Title } from '@patternfly/react-core';
+import { FormikProps, FormikValues } from 'formik';
+import * as _ from 'lodash';
+import { Trans, useTranslation } from 'react-i18next';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import {
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+} from '@console/internal/components/factory/modal';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { InputField } from '@console/shared';
+import BindableServices from './BindableServices';
+
+export type CreateServiceBindingFormProps = {
+  resource: K8sResourceKind;
+  initialApplication: string;
+  cancel?: () => void;
+};
+
+const CreateServiceBindingForm: React.FC<FormikProps<FormikValues> &
+  CreateServiceBindingFormProps> = ({
+  resource,
+  handleSubmit,
+  isSubmitting,
+  cancel,
+  status,
+  dirty,
+  errors,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <form onSubmit={handleSubmit} className="modal-content modal-content--no-inner-scroll">
+      <ModalTitle>{t('console-app~Create Service Binding')}</ModalTitle>
+      <ModalBody>
+        <Title headingLevel="h2" size="md" className="co-m-form-row">
+          <Trans ns="console-app">Select a service you want to create a binding with</Trans>
```

Comment:
Here we can use `t` function for translations as it is just a static string.
```suggestion
          {t('console-app~Select a service you want to create a binding with')}
```